### PR TITLE
Add GS1 8110 Coupon Credential.

### PR DESCRIPTION
**Pull request by BigBlueHat**
I'll follow this PR up with another one later containing the vocabulary documentation and a README. The goal for the moment is just to get this added here and to the [VC Playground](https://vcplayground.org/).

Cheers!
🎩

### CODE REVIEW

File: credentials/gs1-8110-coupon/credential.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "type": ["VerifiableCredential", "GS18110CouponCredential"],
+  "issuer": {
+    "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
+    "name": "Big Retail Store",
+    "url": "https://example.com",
+    "image": "https://vcplayground.org/examples/gs1-8110-coupon/logo.png"
+  },
+  "name": "Digital coupon: Buy one, get one free 16 oz coffees.",
+  "issuanceDate": "2023-06-04T00:00:00Z",
+  "expirationDate": "2023-08-01T08:00:00Z",
```
Comment (msporny) [inline credentials/gs1-8110-coupon/credential.json:15]:
```suggestion
  "expirationDate": "2024-08-01T08:00:00Z",
```

Comment (BigBlueHat) [reply credentials/gs1-8110-coupon/credential.json]: I just removed it entirely to avoid the issue.

**Comment by JSAssassin** [comment]
@BigBlueHat The PR needs a changelog entry.

File: credentials/gs1-8110-coupon/credential.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://contexts.vcplayground.org/examples/gs1-8110-coupon/v1.json"
+  ],
+  "type": ["VerifiableCredential", "GS18110CouponCredential"],
+  "issuer": {
+    "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
+    "name": "Big Retail Store",
```
Comment (davidlehn) [inline credentials/gs1-8110-coupon/credential.json:9]: Should this match "BIG MEGA" logo?

File: credentials/gs1-8110-coupon/image.svg
Comment (davidlehn) [inline credentials/gs1-8110-coupon/image.svg:1]: Expires is in the past.  And the credential doesn't have an expiration.

File: credentials/gs1-8110-coupon/credential.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "image": "https://vcplayground.org/examples/gs1-8110-coupon/logo.png"
+  },
+  "name": "Digital coupon: Buy one, get one free 16 oz coffees.",
+  "issuanceDate": "2023-06-04T00:00:00Z",
+  "image": "https://vcplayground.org/examples/gs1-8110-coupon/image.png",
+  "credentialSubject": {
+    "id": "did:key:123",
+    "clippedCoupon": {
+      "type": "CouponOffer",
+      "upc": "5-12345-000-10-7",
```
Comment (davidlehn) [inline credentials/gs1-8110-coupon/credential.json:20]: Term not defined.

**Comment by BigBlueHat** [comment]
> @BigBlueHat The PR needs a changelog entry.

Added.

File: credentials/gs1-8110-coupon/credential.json (hunk 3 above)
Comment (BigBlueHat) [reply credentials/gs1-8110-coupon/credential.json]: Fixed. That should have been `gtin` anyhow--which subsumed the older UPC format sometime ago (though "UPC" is still used to reference GTIN-12's apparently...hence my confusion).

**Comment by BigBlueHat** [comment]
This should be ready to go. I'm working on the vocabulary documentation now. I'll submit that as a separate PR.

**Pull request merged**